docs/specviz/displaying.rst

```diff
@@ -29,7 +29,68 @@ Data can be selected and de-selected by clicking on the "hammer and screwdriver"
 Pan/Zoom
 ========
 
-Coming soon
+Included with the Specviz tool are multiple manners to change your view/perspective on the data displayed, depending on your environment:
+
+Interactive Pan/Zoom (Desktop App interface)
+--------------------------------------------
+You can find the following Pan/Zoom tools available in the tool drawer of the viewer:
+
+2D Bidirectional Pan/Zoom
+^^^^^^^^^^^^^^^^^^^^^^^^^
+The 2D Pan/Zoom Icon allows you to zoom using the scroll wheel. The window will zoom into the area around your cursor:
+
+.. image:: ../img/SpecViz_2D_Zoom.gif
+    :alt: Zooming in Specviz with the Scroll Wheel
+
+To pan, simply click and drag the window
+
+Horizontal/Vertical Zoom
+^^^^^^^^^^^^^^^^^^^^^^^^
+The Horizontal and Vertical Zoom tools allow you to zoom along each axis, while locking the other. Zoom by scrolling
+
+.. note::
+    A gif here showing how these tools manipulate the view, similar to above may be useful here
```

Comment:
```suggestion
The Horizontal and Vertical Zoom tools allow you to zoom along each axis, while locking the other. You can also zoom by scrolling.
```
I would remove the note, this seems like something that should stay internal and should not be on the user facing documentation.

Reply:
Hmm... I was following guidance from @orifox about that note. Maybe we should rediscuss what to do?

Reply:
Brought this up at the tag up. Will make a confluence page to talk about all possible suggestions